### ISSUE TITLE
Ashfern generation issues

### ISSUE THREAD

**Issue by TripleThreeT** [Tslat/Advent-Of-Ascension#4505]
**Mod Version**
3.7.15
**Describe the bug**
When teleporting to a precasian volcano or deadlands, the ashfern would generate in the close chunks I spawn in, but looking at the rest of the biome the ashfern wouldn't generate at all. I did this a few times with the same results. Here's an image of one. This is the same biome, but one in a chunk I spawned in compared to the others which were slightly farther away.

![2024-12-29_07 44 40](https://github.com/user-attachments/assets/3fda7c18-93d2-4a11-9018-7d185d36d8ce)
![2024-12-29_07 45 08](https://github.com/user-attachments/assets/5a068f82-6d3b-47d4-b56f-452912c71dd7)
![2024-12-29_07 45 23](https://github.com/user-attachments/assets/c9adb7af-2604-4eee-967a-ec9a27a2f87b)

**Comment by Tslat** [comment]
I can't see how this would ever be anything but RNG lol

There's literally nothing in the spawning that could cause this unless it's some completely obscure vanilla bug I don't even have a clue on starting to track down

**Comment by TripleThreeT** [comment]
I don't believe it's random. At least for me that is. Tested again today in another world and got the same results. Then I tried to flying to a precasian deadlands without teleporting. Once I got to the deadlands there were no ashferns to be found at all.

**Comment by Tslat** [comment]
Hmm

That should be easily confirmable then

Let me try some things

**Comment by Tslat** [comment]
Damn this does indeed seem to be happening lol
That's absolutely bizarre

Tested on seed -6949884712928688730

**Comment by Tslat** [comment]
Hmm...

![Image](https://github.com/user-attachments/assets/edc4aba6-c6b4-4551-bbb3-dcc986544ad8)

**Comment by Tslat** [comment]
![Image](https://github.com/user-attachments/assets/95da25a7-c4da-4885-8c7c-28ba6cd607f4)
